translator-files/statistics

```diff
@@ -0,0 +1,475 @@
+pt:
+pt/Compiling.po
```

Comment:
This or status files in general shouldn’t go into the repo (we don’t want it to be logged/created and committed all the time since it just shows a status; nothing directly essential to the page creation). In future we might want to create a note card on the tracking board which logs the status of translations.

Reply:
The idea was to offer an up-to-date overview of the status without having to load the files anywhere or run a script. If it can be shown elsewhere (and in a more visually appealing way) I'm all for it.

Reply:
Something in GH actions probably?

I wouldn’t like to have it in the repo (online) personally. However offline should be ok. 
We could add this file to a [.gitignore](https://www.atlassian.com/git/tutorials/saving-changes/gitignore)?

Reply:
If the po4a scripts are run via gh actions, then this stats file gets produced as a side effect, nothing extra needs to be done. Sure, we can add it to gitignore.

Reply:
I was under the impression that including something in .gitignore meant that it wasn't tracked any more, but still "existed". But if it won't be visible on the repo, there's not really any point in having it - there are other ways to view the translation status offline. This was simply to offer an overview without having to clone the repo or do anything.

Reply:
> This was simply to offer an overview without having to clone the repo or do anything.

Yes that would be nice; however I don’t like the idea of having volatile files on the repo. I‘d rather not add it to the repo but focus on alternatives via GH Actions.

Reply:
No problem - so we can close this then

Reply:
See https://github.com/jamulussoftware/jamuluswebsite/issues/554 and my comment on DC element.